components/executors/dind/patches/executor.ConfigMap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: executor-config
+# Refer to https://docs.sourcegraph.com/admin/executors/deploy_executors_binary#step-2-setup-environment-variables on how to populate these variables
+data:
+  EXECUTOR_FRONTEND_URL: "http://sourcegraph.example.com"
+  EXECUTOR_FRONTEND_PASSWORD: "our-shared-secret"
```

Comment:
This would be better off in a secret instead of a config map value

Reply:
I added a `secretGenerator` for the password and reference this secret in the `Deployment` manifest